### ISSUE TITLE
Feature request - autoclean on interval

### ISSUE THREAD

**Issue by barronen1** [peno64/script.ezmaintenanceplus#3]
It is possible for you to add options to AutoCleanup, not only on Start?
Like daily/weekly/etc

example for that feature is in OpenWizard addon

thanks!

**Comment by peno64** [comment]
I looked at the code in OpenWizard and I indeed see an implementation for it, but I really wonder it also works there...

**Comment by barronen1** [comment]
You can do better implementation 😉

**Comment by peno64** [comment]
@barronen1 
I just released a new version. See https://github.com/peno64/script.ezmaintenanceplus/releases
It is not yet in my repo. I would like to have a test period first.

**Comment by TwilightMercy** [comment]
Thank you very much

**Comment by TwilightMercy** [comment]
I have it configured every 1 day like in picture
But it still cleans everytime I restart Kodi

![image](https://user-images.githubusercontent.com/29895680/212530077-ec308bbe-9b64-4d91-b390-33aac119b23a.png)

**Comment by peno64** [comment]
Then uncheck autoclean cache on startup...

**Comment by TwilightMercy** [comment]
Done, still clears every startup for me

![image](https://user-images.githubusercontent.com/29895680/212531830-3dd28b81-712d-4cc0-a2b4-4d579ec08bad.png)

**Comment by peno64** [comment]
Not here and not according to the code...

I see you are using a special kind of skin. Who knows it's that. Can you try with the standard kodi skin (Estuary)

**Comment by TwilightMercy** [comment]
Yeah it was from phone sorry, tested now in Windows PC, same results. configuration:

![image](https://user-images.githubusercontent.com/29895680/212584750-2eb456cb-41a0-49eb-9e80-2bbb8a939cb5.png)

**Comment by peno64** [comment]
This is really really strange. Can you try with attached zip and then provide me the kodi.log file. It will log extra information such that I have more information.

[script.ezmaintenanceplus-2023.01.09.0.zip](https://github.com/peno64/script.ezmaintenanceplus/files/10428259/script.ezmaintenanceplus-2023.01.09.0.zip)

**Comment by TwilightMercy** [comment]
Sure. With logging debug on?

**Comment by peno64** [comment]
no debug logging is not needed

**Comment by peno64** [comment]
@barronen1 
Any progress with making/sending a kodi log? EzMaintenance kan make one for you and you can upload it such that you only need to provide a link here.

**Comment by TwilightMercy** [comment]
Done.

Current configuration:

![image](https://user-images.githubusercontent.com/29895680/213195645-c76e8617-98e3-4591-a7b1-a0e16aa9c4dd.png)


kodi.log:
https://pastebin.com/UALuVwmR

kodi.old.log:
https://pastebin.com/1UhMf68z

**Comment by peno64** [comment]
If you look in thtis kodi.log file:

2023-01-18 16:20:58.010 T:16676    info <general>: ezmaintenanceplus: auto_clean = false

2023-01-18 16:21:01.039 T:16676    info <general>: ezmaintenanceplus: No auto cleaning - auto_clean = false

So there is  really no auto-clean

How do you conclude that an auto-clean is done?

Is it not a cleanup of packages and thumbnails that is done ? That is something different and is not part of that checkbox. Has never been.

![image](https://user-images.githubusercontent.com/32430988/213278377-73293864-d212-4e9c-a4ed-d1a90c15e4e3.png)

If you don't want this to be done, put the file size to their maximum values.

**Comment by TwilightMercy** [comment]
Every Kodi restart I get the following notification:

![image](https://user-images.githubusercontent.com/29895680/213284203-5d661d41-bc14-4255-b837-4189ba8b44e3.png)


And if “Show Stats” is off, “so maintenance Complete” text.

**Comment by peno64** [comment]
Yes.That is clear packages, *not* auto cleanup.

It depends on this setting and  this setting only and is *only* done at startup or if you do it manually:

![image](https://user-images.githubusercontent.com/32430988/213314404-68609908-42c6-464c-a978-f6d8f865d19e.png)

**Comment by peno64** [comment]
And I think you see this because you enabled this setting:

![image](https://user-images.githubusercontent.com/32430988/213314566-d97f9393-ee57-42d0-8568-572751122e87.png)

**Comment by TwilightMercy** [comment]
Even if “Show Stats at Startup” is off, “maintenance Complete” notification appears. 

So can autoclean every number of days setting can be for everything? (Packages, Thumbnails, Cache)

**Comment by peno64** [comment]
I don't thtink it is a good idea to clean thumbnais every x days. And packages... I don't know
